static/scss/answers/cards/product-prominentvideo.scss

```diff
@@ -8,19 +8,22 @@
   width: 100%;
   height: 100%;
 
+  &-body
+  {
+    padding: calc(var(--yxt-base-spacing) * 3/4) var(--yxt-base-spacing);
+  }
+
   &-header
```

Comment:
we may not need this anymore if that div with classname `HitchhikerProductProminentVideo-header` is replaced to `-body` instead

Reply:
you're right! i'll remove

Reply:
updated!